Kickstarter-iOS/Views/LoadingButton.swift

```diff
@@ -0,0 +1,115 @@
+import Library
+import Prelude
+import UIKit
+
+final class LoadingButton: UIButton {
+  // MARK: - Properties
+
+  private lazy var activityIndicator: UIActivityIndicatorView = {
+    UIActivityIndicatorView(style: .white)
+      |> \.hidesWhenStopped .~ true
```

Comment:
This is the magic `prop` that does not require us to add/remove it from the view hierarchy...it will automatically show/hide the component depending on the `isAnimating` property.